_data/accessibility-tests/footer.yml

```diff
@@ -0,0 +1,97 @@
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.10
+  - summary: Footer icons and non-text content meet color contrast requirements. 
+    summary_additional: When you use a color contrast checker to view the footer, contrast between the icon or other non-text elements and adjacent colors is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
+  - summary: Links provide direct location description.
+    summary_additional: You can read the text of any link in the footer to understand where the link will take you.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.4
+  - summary: Form field instructions are visible.
+    summary_additional: When viewing a form field in the big footer, instructions for entering text are visible.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 3.3.2
+# Zoom/screen magnification tests
+  - summary: Footer remains visible and functional when screen is magnified.
+    summary_additional: When you zoom in to 200%, you can still see footer text and it doesn't overlap with or get cut off by any other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.4
+# Keyboard navigation tests
+  - summary: Keyboard navigates through all elements in the footer.
+    summary_additional: When you navigate through the footer with a keyboard, you can interact with every element inside the footer.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.1
+  - summary: Footer does not trap keyboard focus.
+    summary_additional: When you navigate with a keyboard, you can easily move into and out of the footer.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.1.2
+  - summary: Focus indicator is clearly visible.
+    summary_additional: When you use a keyboard to focus on footer elements, there will be a visible outline or other clear indication of focus. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.1
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader announces image alt text.
+    summary_additional: When you navigate through the footer using a screen reader, you hear descriptions for any important images or graphics.
+    test_status: conditional
+    test_type: screen_reader
+    version_tested: 3.8.1
+    wcag_criterion: 1.1.1
+  - summary: Screen reader announces footer navigation or region.
+    summary_additional: When using a screen reader to acces the footer, you hear "footer navigation, navigation landmark (or region in JAWS). You can also use shortcut keys in the screen reader to navigate the footer. 
```

Comment:
Definitely a typo lol. And yes, I think quotes around the words make sense. I am wondering what you all hear with VO, as what if it does not say any of these mentioned options above? Then I feel like it may need to be generalized potentially.

Reply:
In VO, I hear "Footer navigation, navigation" around the link list. I also hear "footer" on the outer region of the component. 

**question (non-blocking):** Is the test intended for just the navigation region or the whole component? Maybe both? Wondering if it needs to be rephrased from:

**Screen reader announces footer navigation or region.** When using a screen reader to acces the footer, you hear "footer navigation, navigation landmark (or region in JAWS). You can also use shortcut keys in the screen reader to navigate the footer. 

to

**Screen reader announces footer navigation and region.** When using a screen reader to navigate the footer, you hear "footer navigation" when focusing on the link list and "footer" when focusing on the outer region of the component.

Reply:
I agree. I like the edit.

Reply:
Resolved in d08c1f56